From: Dave Airlie <airlied at redhat.com>

We are getting inconsistent methods for endian detection (same answer when
it works, just doesn't work on some platforms) depending on whether __GLIBC__
is defined, which of course depends on include ordering before p_config.h

Just make p_config.h include limits.h to solve this.

Signed-off-by: Dave Airlie <airlied at redhat.com>
---
 src/gallium/include/pipe/p_config.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/gallium/include/pipe/p_config.h b/src/gallium/include/pipe/p_config.h
index 1818c8b..40f6f2b 100644
--- a/src/gallium/include/pipe/p_config.h
+++ b/src/gallium/include/pipe/p_config.h
@@ -46,7 +46,7 @@
 #ifndef P_CONFIG_H_
 #define P_CONFIG_H_
 
-
+#include <limits.h>
 /*
  * Compiler
  */
-- 
1.7.4.4
